### DISCUSSION TITLE
Not getting test/skip reports

### DISCUSSION THREAD

**Discussion by MichaelDLS** [webdriverio/webdriverio#8648] [Q&A]
I use to get the report with test suite along with the test being run and now I can't figure out how to get that output. I now get only 1 test report when there are 2 test in my suite.

<img width="1353" alt="Screen Shot 2022-08-04 at 6 59 06 PM" src="https://user-images.githubusercontent.com/54923635/183001201-5b74b936-a1e7-4d67-bde3-adf2e7362e4e.png">

**Comment by christian-bromann** [comment]
Not sure what you are exactly asking here but what you see in the bottom is the result of all test spec files. Apparently you are running 1 spec file containing 2 tests from which one is skipped. It would show skip in there if the whole file would get skipped.
